samples/04_gis_analysts_data_scientists/forecasting_monthly_rainfall_in_California_using_deeplearning_timeseries_model_from_arcgis_learn.ipynb

```diff
@@ -0,0 +1,2195 @@
+{
```

Comment:
<p>So there are 480 data points, but is it for 20 years, or for 1980-2018, as seen in the table below (20 years of date or 38)? Also, ,you mention 13 years of data being used for training, and 1 year used for testing, which, if there are 20 years of data, excludes 6 years of data? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/900/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='30'/>